go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/coinbase/rosetta-sdk-go v0.6.10
 	github.com/confio/ics23/go v0.6.6
+	github.com/cosmos/cosmos-sdk/db v0.0.0-20210831080937-2c31451a55b5
```

Comment:
We should ideally point to releases. Is that expected in the future?

Reply:
I can add a tag once this is ready to merge

Reply:
I think this would have to be a tag on this repository. Should I make a `db/v0.1.0` tag on this PR and point to that?

Reply:
(Resolved this with a `=> ./db` replace directive)